CRM/ACL/BAO/ACL.php

```diff
@@ -476,8 +476,7 @@ public static function group(
     if (!empty($acls)) {
       $aclKeys = array_keys($acls);
       $aclKeys = implode(',', $aclKeys);
-
-      $cacheKey = CRM_Utils_Cache::cleanKey("$type-$tableName-$aclKeys");
+      $cacheKey = CRM_Utils_Cache::cleanKey("{$tableName}-{$type}-{$aclKeys}");
```

Comment:
I think this change should be removed - looking at the original PR the issue was that type was missing from the cacheKey - that has been fixed & I suspect this line it no longer needed (& there more code there is to understand the harder it is to take people along that)

I would suggest you might remove this & do the commit squash & do a force push

Reply:
@dwoods-encircle can you take care of this?

Reply:
That's been done @colemanw